sql/updates/world/master/2025_03_11_00_Darkshore.sql

```diff
@@ -0,0 +1,544 @@
+-- quest_template_addon
+INSERT INTO `world`.`quest_template_addon` (`ID`, `PrevQuestID`, `NextQuestId`, `ExclusiveGroup`) VALUES (13583, 13582, 13586, -13581);
+INSERT INTO `world`.`quest_template_addon` (`ID`, `PrevQuestID`) VALUES (13902, 13588);
+UPDATE `world`.`quest_template_addon` SET `PrevQuestID`=13940 where `ID`=13588;
+UPDATE `world`.`quest_template_addon` SET `NextQuestId`=13586, `ExclusiveGroup`=-13581 WHERE `ID`=13581;
+UPDATE `world`.`quest_template_addon` SET `NextQuestId`=13586, `ExclusiveGroup`=-13581 WHERE `ID`=13585;
+
+-- creature_template_model
+UPDATE `world`.`creature_template_model` SET `CreatureDisplayID`=5494 WHERE  `CreatureID`=32985 AND `Idx`=0;
+UPDATE `world`.`creature_template_model` SET `CreatureDisplayID`=5494 WHERE  `CreatureID`=32985 AND `Idx`=1;
+
+
+-- creature_loot_template
+
+INSERT INTO `world`.`creature_loot_template` (`Entry`, `Item`, `Chance`, `QuestRequired`) VALUES
+(32935, 44863, 40, 1), -- Corrupted Tide Crawler Flesh
+(32935, 2674, 41, 0), -- Crawler Meat
+(32935, 2675, 39, 0), -- Crawler Claw
+(32935, 774, 0.4, 0), -- Malachite
+(32935, 818, 0.3, 0), -- Tigerseye
+(32935, 727, 0.14, 0), -- Notched Shortsword
+(32935, 14086, 0.13, 0), -- Beaded Sandals
+(32935, 14089, 0.13, 0), -- Beaded Gloves
+(32935, 8180, 0.13, 0), -- Hunting Bow
+(32935, 8178, 0.13, 0), -- Training Sword
+(32935, 4562, 0.13, 0), -- Severing Axe
+(32935, 15473, 0.12, 0), -- Charger's Boots
+(32935, 4672, 0.12, 0), -- Ancestral Belt
+(32935, 2140, 0.12, 0), -- Carving Knife
```

Comment:
Loot is not that easy. Blizz use reference tables instead of adding the same items multiple times to every creature.